### PR TITLE
Use '[option...]' not '[[ options ]]' in text

### PR DESCRIPTION
**Pull request by richsalz**
Looks more like manpage format. :)
Also remove `{{..}}` notation and rewrite around it.

Related to commentary in #16325

### CODE REVIEW

**Comment by levitte** [comment]
I would prefer to have space separation, i.e. `[ options ... ]` rather than `[options...]`

**Comment by InfoHunter** [comment]
> I would prefer to have space separation, i.e. `[ options ... ]` rather than `[options...]`

Any reason behind that preference?

**Comment by levitte** [comment]
> > I would prefer to have space separation, i.e. `[ options ... ]` rather than `[options...]`
> 
> Any reason behind that preference?

Readability when reading quickly, especially in terms of operating systems where `[` and `]` are significant parts of actual command line arguments.

**Comment by richsalz** [comment]
There are no spaces around the square brackets in any of the manpages. I think it should look the same.  Looking at https://vmssoftware.com/docs/VSI_DCL_DICT_VOL_I.pdf VMS doesn't use spaces either.

**Comment by levitte** [comment]
> Looking at https://vmssoftware.com/docs/VSI_DCL_DICT_VOL_I.pdf VMS doesn't use spaces either.

Yeah, and I've mentioned that I dislike that...

Oh well, I won't block this PR because of this.

File: INSTALL.md [hunk 1]
```diff
@@ -1209,21 +1200,21 @@ directory and invoking the configuration commands from there.
...
-    $ /PATH/TO/OPENSSL/SOURCE/Configure [[ options ]]
+    $ /PATH/TO/OPENSSL/SOURCE/Configure [options...]
 
 #### OpenVMS example
 
     $ set default sys$login:
     $ create/dir [.tmp.openssl-build]
     $ set default [.tmp.openssl-build]
-    $ perl D:[PATH.TO.OPENSSL.SOURCE]Configure [[ options ]]
+    $ perl D:[PATH.TO.OPENSSL.SOURCE]Configure [options...]
```
Comment (levitte) [inline INSTALL.md:1210]: So just FYI, `[options...]` is correct syntax on VMS if you want to look at a directory tree, i.e. the directory `[options]` and all its subdirectories...

Comment (richsalz) [reply INSTALL.md:1210]: I worked on VMS decades ago.  Who had a "whoami.dcl" command and by default we included RCS information at the end, so you can imagine what happened with the "$Log$" entries :)

**Review by t8m — APPROVED** [review]

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by richsalz** [comment]
The failing CI tests are not the result of this PR which just changes documentation and echo messages.  I think this can be merged.

**Comment by richsalz** [comment]
The window is closing, can this approved doc fix be merged?

**Comment by richsalz** [comment]
Can someone add the post-3.0 tag since the repo is now frozen for the 3.0 release and this didn't make it?

**Comment by beldmit** [comment]
Hopefully done

**Comment by richsalz** [comment]
Perhaps I spoke too soon.  Is 3.0 frozen?  CAN THIS GO IN?

**Comment by t8m** [comment]
OTC needs to look if this is OK to go in 3.0.x

**Comment by levitte** [comment]
> CAN THIS GO IN?

The 'approval: done' and 'approval: ready to merge' labels should say plenty.

**Comment by t8m** [comment]
I think there is no dispute about master. The question is the 3.0 branch - is this a bug fix? It's unclear to me.

**Comment by levitte** [comment]
Merged to master

a4ffb33ea8b7bcf04b8181dafce7ac512081d0ab Use '[option...]' not '[[ options ]]' in text

**Comment by levitte** [comment]
Cherry-picking to the 3.0 branch pending on OTC decision.

**Comment by richsalz** [comment]
I had wanted this to go into 3.0.  If the project thinks its a bugfix, cherry-picking the commit should just work, but someone else should take that on.

**Comment by levitte** [comment]
> I had wanted this to go into 3.0. If the project thinks its a bugfix, cherry-picking the commit should just work, but someone else should take that on.

Someone else *will* take that on, just as someone took on merging this into master.  Cherry-picking doesn't require a whole new PR

**Pull request closed without merging**